README.md

```diff
@@ -1,106 +1,16 @@
-
-```
-/openDS             <!-- introduction and material relevant to the specification as a whole -->
-│   README.md
-│   positioning-opends.md
-|   serialization.md
-|   use-cases.md
-|   ... ...    
-│
-+---/api-spec       <!-- the api specification -->
-|       api-intro.md
-│       etc. ...
-|
-+---/data-model     <!-- the data model -->
-│       data-model-intro.md
-│       etc. ...
-|
-+---ods-ontology    <!-- the ontology -->
-|       ods-ont-intro.md
-|       etc. ...
-│
-+---/faqs           <!-- frequently asked questions and answers -->
-│       faq.md
-|       etc. ...
-│
-+---/images         <!-- images used throughout -->
-|       etc. ...
-|
-+---/templates      <!-- markdown templates for definitions and issues -->
-|       etc. ...
-|
-|
-
-```
-
-## Way of working
-
-*To be completed.*
-
-## Convergence and global participation
-
-Linking third-party/extended specimen data together, as openDS proposes requires many stakeholders to adopt the standard. This is especially the case to integrate data at multiple levels: specimen, collection types, species, etc. and cross-discipline to geology, genomics, etc. Thus, an eventual standard needs global collaboration of all players in the landscape converging towards a single specification as far as possible. Fragmentation across countries is not helpful considering that collections-based science is global in nature and the actors practising it are spread worldwide.
-
-The openDS initiative is open to global participation and supports movement towards a collaborative global process leading to a standard on a converged Digital Specimen (DS) / Extended Specimen (ES) concept; for example, under the umbrella of the [Alliance for Biodiversity Knowledge](https://www.biodiversityinformatics.org/). Such a process should have engagement from all stakeholders having interest in specimens/samples (not just biodiversity ones).
-
-## Use cases
-
-DS/ES and openDS must simultaneously simplify and answer the needs of all actors in the landscape – publishers, aggregators, relevant authority repositories and downstream users (consumers) of specimen data.
-
-This page of [use cases](Use-cases.md) explains some of the scenarios of use envisaged for openDS.
-
-## Resources
-
-*To be added.*
-
-
-END.
+As of 2024, DiSSCo is supported by different EU grants. We have a dedicated development team and other collaborators working towards a robust specification. You can see some of the implementations in our current [Sandbox](https://sandbox.dissco.tech/). 
 
+Linking third-party/extended specimen data together, as openDS proposes, requires many stakeholders to test and adopt the specification. This is especially the case to integrate data at multiple levels: specimen, collection types, species, etc., and cross-discipline to geology, genomics, etc. This necessitates global collaboration among all players in the landscape converging towards a single specification as far as possible. 
```

Comment:
The etc. in between makes it a bit fuzzy to read, maybe this is for keeping geology apart from biology, but then write it a bit differently.

Maybe change to: This is especially the case for integrating data at multiple levels, like: specimen, collection types, species, genomics, and cross-discipline to geology, etc.

Reply:
good suggestion. updated the line.